src/models/data.py

```diff
@@ -611,9 +647,25 @@ def _calculate_static(self, num_instances: int) -> np.ndarray:
                 )
 
                 static_np = (
-                    static_np - self.static_normalizing_array["mean"]
-                ) / self.static_normalizing_array["std"]
+                    (static_np - self.static_normalizing_array["mean"])
+                    / [
+                        s if s != 0 else 1e-10
```

Comment:
is this the more correct way to replace the zero std ? I think it should be ONE or the OTHER

Reply:
I would make the std 1 if it is zero - see the point above